packages/datepicker/src/css/index.ts

```diff
@@ -109,7 +109,10 @@ export default {
     backgroundColor: 'transparent',
     border: 'none',
     [`&:focus`]: {
-      outline: 'none'
+      borderRadius: '50%',
+      border: `2px solid ${colorsBlue[6]}`,
+      background: colorsBlue[2],
+      color: colorsTextIcon.highOnLight
```

Comment:
```suggestion
      color: colorsTextIcon.highOnLight,
      outline: 'none'
```

@EdwardIrby Since we have a custom styles for focus state the default focus styles can be removed.